pkg/sentry/socket/netfilter/netfilter.go

```diff
@@ -51,7 +53,7 @@ func GetInfo(t *kernel.Task, ep tcpip.Endpoint, outPtr usermem.Addr) (linux.IPTG
 	}
 
 	// Find the appropriate table.
-	table, err := findTable(ep, info.TableName())
+	table, err := findTable(ep, info.Name.String())
```

Comment:
Optional: Instead of taking a string, it would be type-safer to take a TableName and call name.String() inside findTable method.

Same for converNetstackToBinary, and other places we are calling String() in the function call.